### DISCUSSION TITLE
After updating react-hook-form to 6.12.2 getting Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?

### DISCUSSION THREAD

**Discussion by yaroslav-prokipchyn** [react-hook-form/react-hook-form#3660] [General]
Hello after updating react-hook-form from ^6.4.1 to 6.12.2 I start getting a warning 
** Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?**
in chrome console when instantiating this code 
`<Controller
          as={Select}
          control={control}
          name={'status'}
          rules={{ required: true }}
      ....
        />`
note this as={Select}  select is functional component, wrapper over reasct-supper-select.

I don't use **ref** explicitly and believe this can be incorrect code in the lib.
Tell me please what can be the reason for this warning.

**Comment by bluebill1049** [comment]
which version of react-select are you using?

https://codesandbox.io/s/react-hook-form-focus-74ecu?file=/src/index.js
